server/middleware/validationMiddleware.ts

```diff
@@ -86,7 +86,8 @@ export const validate = (schema: z.ZodTypeAny | SchemaFactory): RequestHandler =
       )
     }
     req.flash(FLASH_KEY__VALIDATION_ERRORS, JSON.stringify(deduplicatedFieldErrors))
-    return res.redirect('back')
+    // Remove any hash from the URL by appending an empty hash string
+    return res.redirect(`${req.baseUrl}#`)
```

Comment:
there are 7 more places where we used `res.redirect('back')`
can we reuse the same function here?

Reply:
This comes from the A/C to highlight/focus the error box when there's been a validation error. If the url ends with an anchor? (#) to a control then that control gets focus instead. `res.redirect('back')` keeps the old # reference and this was the only way I could figure around removing it

Reply:
the comment doesn't seem to mean that tho lol